packages/ember-glimmer/lib/syntax/curly-component.js

```diff
@@ -235,6 +235,10 @@ export class CurlyComponentDefinition extends ComponentDefinition {
     builder.attrs.static('class', 'ember-view');
   }
 
+  getComponentClass() {
```

Comment:
The idea is that static lookup will be the norm or use the dynamic helper, these inbetween dynamic stuff has to be done carefully and is not intended to be long term.